docs/develop/build-a-dApp/build-smart-contracts/evm_extensions.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 3
+---
+
+# EVM Extensions
+
+## Overview
+
+EVM Extensions are precompiled contracts that are built into the Ethereum
+Virtual Machine (EVM).
```

Comment:
I believe a diagram would be good to illustrate the feature. @MalteHerrmann

Reply:
agree but it's not in the scope of work of this document as this is just the overview and not a deep dive